### DISCUSSION TITLE
Question getting the app running

### DISCUSSION THREAD

**Discussion by jeremyharrris** [silviopaganini/nft-market#11] [Q&A] [answered]
Hi, first off this project is awesome!! 
I'm trying to play around with it for practice, but having some issues getting the app ran on my machine. 

When I run npm start I get this error. I'm probably missing something simple, but any help would be great. 

Thanks!

`npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @fluuuid/nft-marketplace@1.1.0 start: `react-scripts start "react-scripts" "start"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @fluuuid/nft-marketplace@1.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jeremyharris/.npm/_logs/2021-07-02T15_52_10_176Z-debug.log`

**Comment by silviopaganini** [comment, marked as answer]
Thanks! 

have you tried to `yarn install` instead?

**Reply by jeremyharrris** [reply]
I tried yarn install, but getting pretty much the same error

**Reply by silviopaganini** [reply]
Remove node_modules, package lock, yarn lock and try to yarn install

**Reply by jeremyharrris** [reply]
Tried removing all of those, now getting...

`yarn run v1.22.10
$ react-scripts start

There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "babel-loader": "8.1.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of babel-loader was detected higher up in the tree:

  /Users/jeremyharris/J Market/nft-market/node_modules/babel-loader (version: 8.2.2) 

Manually installing incompatible versions is known to cause hard-to-debug issues.

If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.

To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "babel-loader" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /Users/jeremyharris/J Market/nft-market/node_modules/babel-loader is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls babel-loader in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed babel-loader.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!`

**Reply by silviopaganini** [reply]
Yeah try following those steps now

**Reply by jeremyharrris** [reply]
Thanks for the quick response, I fixed it! I needed to set webpack to the correct version and the project started to run : )

**Reply by silviopaganini** [reply]
glad to hear it!
